config/example/ir.yaml

```diff
@@ -48,6 +48,14 @@ morph:
         # TCP addresses in 'host:port' format
         - localhost:30333
         - localhost:30334
+      tls_config: # Additional addresses to listen to using TLS setup; must not overlap with `listen` section
+        enabled: false # Additional TLS serving switcher
```

Comment:
We have the same flag for GRPC TLS and Prometheus service, for example, so we better be consistent and allow it everywhere appropriate.

Reply:
agree with consistency, but the point remains

Reply:
Admins find it useful, it requires less editing to quickly temporarily enable/disable some component when needed.

Reply:
toggling is more rare case (not sure if it's actually used at all) than configuring from scratch. As node admin, I'd like to write endpoints/filepaths and that's it, writing additional `enabled: true` is an extra routine to me. If admin forgets to write `enabled`, then TLS isn't served at all even with specified endpoints and certificate - aso pretty strange.

i don't mind to have `enabled` flag, but still for the most dry config without dualities

Reply:
i took it from `neo-go` and also took part in the same discussion with the same agrs once

the same as in the thread above: i agree with both of you so i am OK with any case @roman-khimov, @AnnaShaleva @cthulhu-rider

Reply:
or even @aprasolova, @532910

Reply:
Port config looks like a trash overall, why tls is inside non tls? What is reason to listen multiple ports with the same cert? I believe my opinion doesn't matter, please don't mention me.

Reply:
1. It's a service in the first place, it can have some service-specific settings applicable to _any_ communication channel, that's why `service.tls` scheme.
2. I know about nspcc-dev/neo-go#2901, but that's not the way RPC is configured now, IR can't provide this functionality until NeoGo has it.
3. I don't care much about beliefs, but I care about valid feedback from product users. You're one of them so you'll be asked anyway.

Reply:
I do not use this product, sorry ):

TLS may have a toggle, if it's an option, like in synapse `listeners` or ejabberd `listen` section.

If I'm a user, please, make the config clear to users, like synapse or ejabberd for example.

Reply:
i guess conversation is stuck, therefore, we stick to the current version, for the rest it will be possible to create an issue